src/sentry/rules/conditions/event_attribute.py

```diff
@@ -52,17 +51,8 @@ class MatchType(object):
 ]
 
 
-class FixedTypeaheadInput(forms.TextInput):
-    def __init__(self, choices, *args, **kwargs):
-        super(FixedTypeaheadInput, self).__init__(*args, **kwargs)
-        self.attrs["data-choices"] = json.dumps(choices)
-        self.attrs["class"] = self.attrs.get("class", "") + " typeahead"
-
-
 class EventAttributeForm(forms.Form):
-    attribute = forms.CharField(
-        widget=FixedTypeaheadInput(choices=[{"id": a, "text": a} for a in ATTR_CHOICES])
-    )
+    attribute = forms.ChoiceField((a, a) for a in ATTR_CHOICES)
```

Comment:
Huh, looking at this in prod... it seems it's already a choice field

Reply:
Yeah because it doesn't use this form, when we converted it to react we added some new `form_json` stuff in the backend to define the form shape.